tests/13-report.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+	<testsuite tests="0" failures="0" time="0.000" name="package/name">
```

Comment:
Actually, maybe I can capture the output, if I'll put the captured output into a map when the beginning of the file is being read. I can have a map between the package name and the error. no problems.